docs/specs/schema.yml

```diff
@@ -368,6 +364,33 @@ outputs:
   a.json: |
     {"prop": "value"}
 ---
+# Data-transform's input should have both system and user metadata
+inputs:
+  docfx.yml: |
+    globalMetadata:
+      globalMta: global
+    fileMetadata:
+      fileMta:
+        a.yml: file
+  a.yml: |
+    #YamlMime:TestData
+    prop: value
+    metadata:
+      userMta: user
+  _themes/ContentTemplate/schemas/TestData.schema.json: "{}"
+outputs:
+  a.json: |
+    {
+      "prop": "value",
+      "metadata": {
```

Comment:
this makes me wonder if we should merge metadata to `/metadata` for page schema as well.

Reply:
We can do that when we are comparing page sdp pipeline?

Reply:
yep, i'm wondering too, continue to check with Adam.